.github/workflows/trivy.yml

```diff
@@ -3,7 +3,7 @@
 # separate terms of service, privacy policy, and support
 # documentation.
 
-name: build
+name: Trivy Code Scan
```

Comment:
Whoops, didn't notice this earlier. :-D

Reply:
Yeah build sounded confusing(we had 3 by that name)